packages/serverless/src/apollo/app/merkle-tree.ts

```diff
@@ -57,9 +57,7 @@ const getWitness = publicWrapper(
   MerkleTreeIndexRequest,
   async (_root: unknown, args: TMerkleTreeIndexRequest) => {
     const merkleTreeService = await ModelMerkleTree.load(args.databaseName);
-    return withTransaction((session) =>
```

Comment:
I use Promise.all inside the getWitness method, maybe we can try to run sequential approach

Reply:
Can you show me why you want to use sequential approach in this case? I think these get should be in concurrency because it don't need to wait each other